src/backend/aspen/workflows/nextstrain_run/builder_base.py

```diff
@@ -29,28 +30,36 @@ def load_template(self):
 
     def update_build(self, config):
         build = config["builds"]["aspen"]
-        # TODO - We'll add region/country info to groups soon, but it's not there yet.
-        # region = self.group.region
-        # country = self.group.country
-        # build["region"] = region
-        # build["title"] = build["title"].format(division=division, location=location, country=country, region=region)
-        country = "USA"
-        division = self.group.division
-        location = self.group.location
-        build["country"] = country
-        build["division"] = division
-        build["location"] = location
+
+        location = self.group.default_tree_location
+        # Make a shortcut to decide whether this is a location vs division vs country level build
+        if not location.division:
+            self.tree_build_level = "country"
+        elif not location.location:
+            self.tree_build_level = "division"
```

Comment:
@danrlu here we're stashing a tree build level as `location` or `division` or `country` so we can use that in `build_config.py`